src/jest.res

```diff
@@ -471,6 +471,23 @@ module Jest = {
     }
   }
   @val external useRealTimers: unit => unit = "jest.useRealTimers"
+  @val external _setSystemTime: Js.Date.t => unit = "jest.setSystemTime"
+  @val external __setSystemTime: unit => unit = "jest.setSystemTime"
+  @val external _setSystemTime: Js.Date.t => unit = "jest.setSystemTime"
+  @val external __setSystemTime: int => unit = "jest.setSystemTime"
+  @val external ___setSystemTime: unit => unit = "jest.setSystemTime"
```

Comment:
There's some duplication here, which should produce "unused" warnings.

Also, while I see that you're just emulating a pattern seen elsewhere (which, in hindsight, I shouldn't have accepted), the naming scheme here with adding more undescores is not very descriptive. Could you change this to `setSystemTimeWithDate` or something instead?

And bonus points if you also fix the other code using this scheme :1st_place_medal:

Reply:
Make sense! Sure, pushing the update 🙌